### ISSUE TITLE
shell: Session menu UI change

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#16173]
We could change the top-right to look like this:

![cockpit-overview-session (copy)](https://user-images.githubusercontent.com/10246/127490160-128dfc7f-d42e-4c05-965b-9e65d5137eb5.png)

I noticed that PF now suggests a filled circle for the heading, wheras the line version is for in-page help icons. This is new. We should update. I've updated it in this mockup. With the outline version that we're currently using, the cog looks out of place, but it fits right in with the solid version.

![Screenshot 2021-07-29 at 14-18-13 PatternFly 4 • Icons](https://user-images.githubusercontent.com/10246/127490468-678cf71b-8777-4428-950a-7ee16aef3251.png)

In context of the rest of the page, it would look like this:

![cockpit-overview-session](https://user-images.githubusercontent.com/10246/127490252-d1ca833c-45c3-46c2-b9e7-e36bff7a141a.png)

---

One problem with the cog: We're using it in the metrics settings. This would mean two cogs right above each other. They'd each have different words, but it'd probably still be a little odd.

Here's a screenshot:

![Screenshot 2021-07-29 at 14-19-39 Overview - garrett Bolt](https://user-images.githubusercontent.com/10246/127490713-6f05b396-fc1b-4156-8a77-f06f8ea47414.png)

And here's a modification to make it show what it would look like even with the stacking:

![Screenshot 2021-07-29 at 14-19-39 Overview - garrett Bolt-mod](https://user-images.githubusercontent.com/10246/127491041-e3765a0b-2f70-41e5-8e49-b54cbca7bd80.png)

Additionally, someone might think the cog is a flower. But that's fine. Whatever, it's the session and settings flower then. :white_flower: :grin: 

If you have a suggestion for something else, PatternFly's icon library is browsable at: https://www.patternfly.org/v4/guidelines/icons/#all-icons

_Originally posted by @garrett in https://github.com/cockpit-project/cockpit/discussions/16154#discussioncomment-1086819_

**Comment by garrett** [comment]
New help icon:

```css
background-image: url("data:image/svg+xml,%3Csvg style='vertical-align:-.125em' fill='currentColor' viewBox='0 0 512 512' aria-hidden='true' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z'/%3E%3C/svg%3E");
```

Cog icon:

```css
background-image: url("data:image/svg+xml,%3Csvg style='vertical-align:-.125em' fill='currentColor' viewBox='0 0 512 512' aria-hidden='true' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M487.4 315.7l-42.6-24.6c4.3-23.2 4.3-47 0-70.2l42.6-24.6c4.9-2.8 7.1-8.6 5.5-14-11.1-35.6-30-67.8-54.7-94.6-3.8-4.1-10-5.1-14.8-2.3L380.8 110c-17.9-15.4-38.5-27.3-60.8-35.1V25.8c0-5.6-3.9-10.5-9.4-11.7-36.7-8.2-74.3-7.8-109.2 0-5.5 1.2-9.4 6.1-9.4 11.7V75c-22.2 7.9-42.8 19.8-60.8 35.1L88.7 85.5c-4.9-2.8-11-1.9-14.8 2.3-24.7 26.7-43.6 58.9-54.7 94.6-1.7 5.4.6 11.2 5.5 14L67.3 221c-4.3 23.2-4.3 47 0 70.2l-42.6 24.6c-4.9 2.8-7.1 8.6-5.5 14 11.1 35.6 30 67.8 54.7 94.6 3.8 4.1 10 5.1 14.8 2.3l42.6-24.6c17.9 15.4 38.5 27.3 60.8 35.1v49.2c0 5.6 3.9 10.5 9.4 11.7 36.7 8.2 74.3 7.8 109.2 0 5.5-1.2 9.4-6.1 9.4-11.7v-49.2c22.2-7.9 42.8-19.8 60.8-35.1l42.6 24.6c4.9 2.8 11 1.9 14.8-2.3 24.7-26.7 43.6-58.9 54.7-94.6 1.5-5.5-.7-11.3-5.6-14.1zM256 336c-44.1 0-80-35.9-80-80s35.9-80 80-80 80 35.9 80 80-35.9 80-80 80z'/%3E%3C/svg%3E");
```

**Comment by martinpitt** [comment]
This is release note worthy, but I included it in #16169
